packages/framework/sveltekit/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "main": "../dist/cjs/servers/sveltekit.cjs",
+  "module": "../dist/esm/servers/sveltekit.ts",
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
⚠ **issue:** Needs to use the `.js` extension for `sveltekit` module export
```suggestion
  "module": "../dist/esm/servers/sveltekit.cjs",
```